standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/metrics/MetricsConstants.java

```diff
@@ -27,6 +27,11 @@
   public static final String COMPACTION_WORKER_CYCLE = "compaction_worker_cycle";
   public static final String OLDEST_OPEN_TXN_ID = "oldest_open_txn_id";
   public static final String OLDEST_OPEN_TXN_AGE = "oldest_open_txn_age_in_sec";
+  // number of aborted txns in TXNS table
+  public static final String NUM_ABORTED_TXNS_IN_TXNS = COMPACTION_STATUS_PREFIX + "aborted_txns_in_txns";
```

Comment:
could this be just NUM_ABORTED_TXNS ?

Reply:
I have to differentiate between 2 metrics:
1. count in TXNs where status='a' (current name: NUM_ABORTED_TXNS_IN_TXNS)
2. total number of aborted txns since hms was started (current name: NUM_ABORTED_WRITE_TXNS but I'd change this to NUM_ABORTED_TXNS)

It would also make sense to change #2 to NUM_TOTAL_ABORTED_TXNS and #1 to NUM_CURRENT_ABORTED_TXNS. Does that make sense to you too?

Reply:
yes, I would go with NUM_TOTAL_ABORTED_TXNS and NUM_ABORTED_TXNS.